File: Scripts_Archive/do_backup.js
const fs = require('fs');
const path = require('path');

// Step 1: Backup
const backupDir = path.join(__dirname, 'backup_v3');
if (!fs.existsSync(backupDir)) {
    fs.mkdirSync(backupDir);
}
['js/main.js', 'index.html', 'css/chat.css', 'js/backup-system.js'].forEach(file => {
    if (fs.existsSync(file)) {
        fs.copyFileSync(file, path.join(backupDir, path.basename(file)));
    }
});
console.log("Backup completed.");

File: Scripts_Archive/do_backup.py
import os, shutil, datetime

base = r'c:\Users\Administrator\Desktop\111'
ts = datetime.datetime.now().strftime('%Y%m%d_%H%M%S')
bdir = os.path.join(base, f'backup_final_{ts}')
os.makedirs(bdir+'/js', exist_ok=True)
os.makedirs(bdir+'/css', exist_ok=True)
shutil.copy(os.path.join(base, 'js/main.js'), bdir+'/js/main.js')
shutil.copy(os.path.join(base, 'index.html'), bdir+'/index.html')
for f in os.listdir(os.path.join(base, 'css')):
    shutil.copy(os.path.join(base, 'css', f), os.path.join(bdir, 'css', f))
print('Backup done:', bdir)
